src/server/scripts/Kalimdor/DireMaul/instance_dire_maul.cpp

```diff
@@ -64,6 +64,9 @@ class instance_dire_maul : public InstanceMapScript
                     }
                     HighborneSummoners.push_back(creature->GetGUID());
                     break;
+                case NPC_ISALIEN:
```

Comment:
where is this guid used? if it's not used anywhere there's no need to declare it